test/e2e-tests/README.md

```diff
@@ -16,17 +16,35 @@ The following enviroment varaibles are required to run the test setup and the te
 - `S3_ENDPOINT` - Endpint of the bucket
 - `IMAGE_REGISTRY_USERNAME` - quay.io username
 - `IMAGE_REGISTRY_PASSWORD` - quay.io password
-- `S3_BUCKET` - Name of S3 bucket that has the model
-- `TARGET_IMAGE_TAGS_JSON` - JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
-- `NAMESPACE` - Cluster namespace used for testing
+
+The following enviroment varaibles are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+
+- `SELF_SIGNED_CERT` - A path on your local machine to a self signed cert to be used in pipeline
+
+
+```bash
+make go-test-setup
+```
 
 ## Run tests locally
+
+The following enviroment varaibles are required to run the tests. If any of these are not set then the tests will not run. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+
+- `S3_BUCKET`	- Name of S3 bucket that contains the models
+- `NAMESPACE`	- Cluster namespace that tests are run in
+- `TARGET_IMAGE_TAGS_JSON`	- JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
+
+The following enviroment varaibles are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+
+- `SELF_SIGNED_CERT` - A path on your local machine to a self signed cert to be used in pipeline
+- `GO` - Custom Go installation path that is not set in your `PATH`
+
 ```bash
 make go-test
 ```
 Set the go binary used for testing that is in your `PATH`
 ```bash
-make GO=go1.19 go-test
+make go-test
```

Comment:
If it stays like it is now, then we can remove the second `make go-test` because now there are two `make go-test` sections. I would put the go binary change back so 2 examples will be showing different things. It even says ```Set the go binary used for testing that is in your `PATH` ``` and now there is no such thing as it was removed.

Reply:
Yeah sorry I think I just forgot to remove the section and just removed the GO part. I will add it back so there are two examples